src/lib/HST/Util/Selectors.hs

```diff
@@ -89,3 +94,96 @@ getPatVarName (S.PTuple _ _ _) = reportFatal
 getPatVarName (S.PList _ _) = reportFatal
   $ Message Error
   $ "Expected variable or wildcard pattern, got list pattern."
+
+-------------------------------------------------------------------------------
+-- Find Identifiers in Modules                                               --
+-------------------------------------------------------------------------------
+-- | Collects all identifiers in a module in a set.
+getIdentifiers :: S.Module a -> Set String
```

Comment:
Since there are variations of `getIdentifiers` for all kinds of nodes, we could move this into a type class. The type class would also allow patterns such as `maybe Set.empty` and `Set.unions .: map` to be abstracted away.